timemachine/cpp/src/nonbonded_common.cu

```diff
@@ -0,0 +1,127 @@
+#include <set>
+#include <stdexcept>
+#include <string>
+
+#include "nonbonded_all_pairs.hpp"
+#include "nonbonded_common.cuh"
+
+#include "fanout_summed_potential.hpp"
+#include "summed_potential.hpp"
+
+#include "set_utils.hpp"
+
+namespace timemachine {
+
+void verify_atom_idxs(int N, const std::vector<int> &atom_idxs, const bool allow_empty) {
```

Comment:
nit:
```suggestion
void verify_atom_idxs(const int N, const std::vector<int> &atom_idxs, const bool allow_empty) {
```

Reply:
Added in [9fa2580](https://github.com/proteneer/timemachine/pull/1053/commits/9fa2580d4e51e70d55ead2eb10fccab9af5567be)